src/io/flutter/inspector/DiagnosticsNode.java

```diff
@@ -694,7 +715,7 @@ private void setCreationLocation(InspectorSourceLocation location) {
     this.location = location;
   }
 
-  public InspectorService getInspectorService() {
+  public InspectorService.ObjectGroup getInspectorService() {
```

Comment:
InspectorService.ObjectGroup is now returned so that we have a service correctly tied
to the lifecycle of this DiagnosticsNode. If the group is disposed then you will get a 
disposed ObjectGroup here which will make all operations short circuit.